guides/hosting/infrastructure/filesystem.md

```diff
@@ -75,12 +75,31 @@ shopware:
     private_local_download_strategy: # Name of the download strategy: php, x-sendfile or x-accel
 ```
 
-Following download strategies are valid:
+The following download strategies are valid:
 
 * `php` (default): A streamed response of content type `application/octet-stream` with binary data
 * `x-sendfile` (Apache only): X-Sendfile allows you to use PHP to instruct the server to send a file to a user, without having to load that file into PHP. You must have the [`mod_xsendfile`](https://github.com/nmaier/mod_xsendfile) Apache module installed.
 * `x-accel` (Nginx only): X-accel allows for internal redirection to a location determined by a header returned from a backend. See the [example configuration](https://www.nginx.com/resources/wiki/start/topics/examples/x-accel/).
 
+## CDN configuration
+
+If your public files are available on a CDN, you can use the following config to serve images and other assets via that CDN.
+
+```yaml
+# <project root>/config/packages/prod/shopware.yml
```

Comment:
Idk, is the path correct? :thinking: 
Because I find the following file:
https://github.com/shopware/shopware/blob/trunk/src/Core/Framework/Resources/config/packages/shopware.yaml
In the prod dir there is no shopware.yaml...